packages/devtools/lib/src/memory/memory.dart

```diff
@@ -418,23 +1057,8 @@ class MemoryScreen extends Screen with SetStateMixin {
       if (ref.valueAsString != null && !ref.valueAsStringIsTruncated) {
         return ref.valueAsString;
       } else {
-        final dynamic result = await serviceManager.service.invoke(
-          _debuggerState.isolateRef?.id,
-          ref.id,
-          'toString',
-          <String>[],
-          disableBreakpoints: true,
-        );
-
-        if (result is ErrorRef) {
-          return '${result.kind} ${result.message}';
-        } else if (result is InstanceRef) {
-          final String str = await _retrieveFullStringValue(result);
-          return str;
-        } else {
-          // TODO: Improve the return value for this case.
-          return null;
-        }
+        // TOOD(terry): Shouldn't happen but want to check - remove later.
+        return 'UNKNOWN BoundField value';
```

Comment:
can we log this to analytics instead of confusing users?

Reply:
Logged to analytics.